src/main/scala/treadle/executable/VerifyOp.scala

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package treadle.executable
+
+import firrtl.WireKind
+import firrtl.ir._
+
+/** This handles the processing of coverage statements
+  *
+  * @param symbol            machine generated symbol for each statement encountered
+  * @param info              source information
+  * @param message           message associated with statement
+  * @param clockTransition   updates clockCount when this is is PosEdge
+  * @param predicate         only increments coverCount if this is true
+  * @param enable            only increments coverCount if this is true
+  */
+case class VerifyOp(
+  symbol:          Symbol,
+  info:            Info,
+  message:         StringLit,
+  clockTransition: ClockTransitionGetter,
+  predicate:       IntExpressionResult,
+  enable:          IntExpressionResult)
+    extends Assigner {
+
+  var clockCount: Long = 0
+  var coverCount: Long = 0
+
+  def run: FuncUnit = {
+    val conditionValue = predicate.apply() > 0
+    val enableValue = enable.apply() > 0
```

Comment:
Is there a reason for computing these unconditionally instead of only after checking the clock transition?

Reply:
Great catch, I'll fix that